README.md

```diff
@@ -1,4 +1,9 @@
-[![Build Status](https://travis-ci.org/kata-containers/proxy.svg?branch=master)](https://travis-ci.org/kata-containers/proxy)
-[![codecov](https://codecov.io/gh/kata-containers/proxy/branch/master/graph/badge.svg)](https://codecov.io/gh/kata-containers/proxy)
-
-# Kata Containers Proxy
+```
```

Comment:
I think you could use larger moo here. :interrobang:

Reply:
What like?

```
MMMMMMMM               MMMMMMMM                                  
M:::::::M             M:::::::M                                  
M::::::::M           M::::::::M                                  
M:::::::::M         M:::::::::M                                  
M::::::::::M       M::::::::::M   ooooooooooo      ooooooooooo   
M:::::::::::M     M:::::::::::M oo:::::::::::oo  oo:::::::::::oo 
M:::::::M::::M   M::::M:::::::Mo:::::::::::::::oo:::::::::::::::o
M::::::M M::::M M::::M M::::::Mo:::::ooooo:::::oo:::::ooooo:::::o
M::::::M  M::::M::::M  M::::::Mo::::o     o::::oo::::o     o::::o
M::::::M   M:::::::M   M::::::Mo::::o     o::::oo::::o     o::::o
M::::::M    M:::::M    M::::::Mo::::o     o::::oo::::o     o::::o
M::::::M     MMMMM     M::::::Mo::::o     o::::oo::::o     o::::o
M::::::M               M::::::Mo:::::ooooo:::::oo:::::ooooo:::::o
M::::::M               M::::::Mo:::::::::::::::oo:::::::::::::::o
M::::::M               M::::::M oo:::::::::::oo  oo:::::::::::oo 
MMMMMMMM               MMMMMMMM   ooooooooooo      ooooooooooo   
```